assets/src/dashboard/app/views/myStories/noResults.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const NoResultsText = styled.p`
+  font-family: ${({ theme }) => theme.fonts.body1.family};
+  font-weight: ${({ theme }) => theme.fonts.body1.weight};
+  font-size: ${({ theme }) => theme.fonts.body1.size};
+  line-height: ${({ theme }) => theme.fonts.body1.lineHeight};
+  letter-spacing: ${({ theme }) => theme.fonts.body1.letterSpacing};
+  color: ${({ theme }) => theme.colors.gray200};
+  margin: 40px 20px;
+`;
+
+const NoResults = ({ typeaheadValue }) => (
+  <NoResultsText>
+    {__("Sorry we couldn't find any results matching", 'web-stories')}{' '}
+    {__('"', 'web-stories')}
+    {typeaheadValue}
```

Comment:
Since `typeaheadValue` is just a string, we can use `sprintf` from the `@wordpress/i18n` package to make this more readable and also more translatable:

```js
sprintf(
  __('Sorry, we couldn\'t find any results matching "%s"', 'web-stories'),
  typeaheadValue
)
```

Reply:
thanks for the info about sprintf, i'd seen it in the includes/Dashboard.php for api endpoint set up but hadn't paid it much attention. I'll swap these out this am.